processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModelBuilder.java

```diff
@@ -434,6 +436,10 @@ private void validateAndSetColumnCompressor(CarbonLoadModel carbonLoadModel)
 
   private void validateAndSetBinaryDecoder(CarbonLoadModel carbonLoadModel) {
     String binaryDecoder = carbonLoadModel.getBinaryDecoder();

```

Comment:
If Binary decoder returns null, it throws exception ?

Reply:
if Binary decoder returns null, it will use default decoder. it will not throws exception. decoder is optional.